### ISSUE TITLE
Learning rate scheduler is working incorrect

### ISSUE THREAD

**Issue by hocop** [leftthomas/SRGAN#94]
While training SRGAN model, learning rate scheduler is used with parameter `step_size=epochs // 2` and `gamma=0.1`.  
From this I suppose `schedulerD.step()` and `schedulerG.step()` to be called at each epoch.  
However, they are called at each training step which means that learning rate is quickly becoming zero after a few training steps.

The SRGAN does not seem to train. But when I put `schedulerD.step()` and `schedulerG.step()` after the epoch end, it started to train.

**Comment by leftthomas** [comment]
@hocop learning rate scheduler is not used in this code.

**Comment by hocop** [comment]
I am very sorry! This was in another repository
